local-interchaintest/src/tests/remote_chain_splitter/remote_chain_splitter.rs

```diff
@@ -0,0 +1,307 @@
+    let atom_denom = test_ctx.get_native_denom().src(GAIA_CHAIN_NAME).get();
+    let uatom_contribution_amount: u128 = 5_000_000_000;
+
+    let split_config: Vec<(String, SplitConfig)> = vec![(
+        atom_denom.clone(),
+        SplitConfig {
+            receivers: BTreeMap::from_iter(vec![
+                (ACC1_ADDRESS_NEUTRON.to_owned(), Decimal::percent(50)),
+                (ACC2_ADDRESS_NEUTRON.to_owned(), Decimal::percent(50)),
+            ]),
+        },
+    )];
+
+    let mut remote_chain_splitter = test_ctx
+        .get_contract("valence_remote_chain_splitter")
+        .unwrap();
+
+    remote_chain_splitter
+        .instantiate(
+            DEFAULT_KEY,
+            serde_json::to_string(&valence_remote_chain_splitter::msg::InstantiateMsg {
+                op_mode_cfg: ContractOperationModeConfig::Permissionless,
+                remote_chain_channel_id: test_ctx
+                    .get_transfer_channels()
+                    .src(NEUTRON_CHAIN_NAME)
+                    .dest(GAIA_CHAIN_NAME)
+                    .get(),
+                remote_chain_connection_id: test_ctx
+                    .get_connections()
+                    .src(NEUTRON_CHAIN_NAME)
+                    .dest(GAIA_CHAIN_NAME)
+                    .get(),
+                denom: atom_denom.clone(),
+                amount: Uint128::from(uatom_contribution_amount),
+                splits: BTreeMap::from_iter(split_config),
+                ica_timeout: Uint64::new(DEFAULT_TIMEOUT),
+                ibc_transfer_timeout: Uint64::new(DEFAULT_TIMEOUT),
+                fallback_address: None,
+            })
+            .unwrap()
+            .as_str(),
+            "valence_remote_chain_splitter",
+            None,
+            "",
+        )
+        .unwrap();
+
+    Ok(ContractInfo {
+        contract_addr: remote_chain_splitter.contract_addr.unwrap(),
+        code_id: remote_chain_splitter.code_id.unwrap(),
+    })
+}
+
+fn fund_remote_chain_splitter(
+    test_ctx: &mut TestContext,
+    remote_chain_splitter: &ContractInfo,
+) -> Result<(), LocalError> {
+    let neutron = test_ctx.get_chain(NEUTRON_CHAIN_NAME);
+
+    neutron
+        .rb
+        .tx(
+            &format!(
+                "tx bank send acc0 {} 1000000untrn",
+                &remote_chain_splitter.contract_addr
+            ),
+            true,
+        )
+        .unwrap();
+
+    Ok(())
+}
+
+/// Tests that the splitter does not advance when there is no relayer available.
+/// Leaves the relayer intact after its execution.
+fn test_remote_chain_splitter_timeout(
+    test_ctx: &mut TestContext,
+    remote_chain_splitter: &ContractInfo,
+) -> Result<(), LocalError> {
+    // Stop the relayer
+    stop_relayer(test_ctx);
+
+    let neutron = test_ctx.get_chain(NEUTRON_CHAIN_NAME);
+
+    let remote_chain_splitter_contract = CosmWasm::new_from_existing(
+        &neutron.rb,
+        None,
+        Some(remote_chain_splitter.code_id),
```

Comment:
In line with the previous one, you can pass None here because it's only used for instantiating, not for executing